scripts/macos-setup-brew.sh

```diff
@@ -16,11 +16,14 @@ done;
 
 echo "Todo:    ${NEEDED_PACKAGES}"
 
+# Remove old python
+# https://discourse.brew.sh/t/python-2-eol-2020/4647
+brew uninstall python@2
```

Comment:
is this a no-op if Python2 is not installed?

Reply:
it will return non-zero which will break bash

Did this
`brew uninstall --force python@2`, which returns 0 even if you've already removed